tests/components/cast/test_media_player.py

```diff
@@ -208,15 +207,6 @@ async def test_stop_discovery_called_on_stop(hass):
 
         stop_discovery.assert_called_once_with(browser)
 
-    with patch(
-        "homeassistant.components.cast.discovery.pychromecast.start_discovery",
-        return_value=browser,
-    ) as start_discovery:
-        # start_discovery should be called again on re-startup
-        await async_setup_cast(hass)
-
-        assert start_discovery.call_count == 1
```

Comment:
This fails after changing according to https://github.com/home-assistant/core/pull/37256#discussion_r449744304
Anyhow, this test seems a bit strange?

Reply:
Yes